src/docs/stan-reference/examples.tex

```diff
@@ -6976,14 +7011,12 @@ \section{Simulating from a Gaussian Process}
 The input data is just the vector of inputs \code{x} and its size
 \code{N}.  Such a model can be used with values of \code{x} evenly
 spaced over some interval in order to plot sample draws of functions
-from a Gaussian process.  The covariance matrix \code{Sigma} is not
-being computed efficiently here; see
-Section~\refsection{fit-gp} for a better approach.
+from a Gaussian process.  
 
 \subsection{Multivariate Inputs}
 
-Only the covariance function's distance computation needs to change in
-moving from a univariate model to a multivariate model.%
+Only the input data needs to change in moving from a univariate model to a
+multivariate model.%
 %
 \footnote{The model is available in the Stan example model repository;
 see \url{http://mc-stan.org/documentation}.}
```

Comment:
Sorry if I'm missing something, but I go this link and I just don't find the GP example models. There are a lot of these footnotes repeated through the chapter.